std/regex/internal/kickstart.d

```diff
@@ -132,6 +132,13 @@ public:
         import std.conv;
         assert(memory.length == 256);
         fChar = uint.max;
+        ulong hash(uint[] tab)
```

Comment:
Truth be told I haven't thought through this at all, just coded something FNV/Bernstain style. Will check FNV-1a. Notice that I use full 64bits of hash so the chance of collision is rather small.